functions.py

```diff
@@ -10,7 +10,7 @@
 from cooking_lists import *
 from questions import *
 
-version = "Cooking Helper v.1"
+version = "Cooking Helper v.2"
```

Comment:
I would suggest renaming the variable into `VERSION` as uppercase is the convention for constants. I actually think that `TITLE` would be a more appropriate name for it since it is the title of every message box and does not only contain the version string in it but the name of the program as well.